### ISSUE TITLE
Error with javascript syntax

### ISSUE THREAD

**Issue by DavidBLynch** [matjaz/aurelia-property-injection#2]
When using the js syntax I'm getting the following error:

```
Unhandled promise rejection Error: ClassActivator is not defined
Error loading http://localhost:9100/my-plugin/index.js
    at Object.<anonymous> (http://localhost:9100/jspm_packages/github/matjaz/aurelia-property-injection@0.2.0/index.js:2:1409)
```

Is there some type of global registration required?

**Comment by matjaz** [comment]
fixed in @40b18a6

**Comment by matjaz** [comment]
Addressed in Microsoft/TypeScript#4589
